tests/conftest.py

```diff
@@ -107,3 +107,13 @@ def GIVEN_registered_account_with_minted_assets(
                 5,
                 asset)]))
     return GIVEN_registered_account
+
+
+@pytest.fixture()
+def GIVEN_registered_transaction(GIVEN_new_domain_id):
```

Comment:
I think `GIVEN_hash_of_registered_transaction` will be better name of the fixture 🤔